packages/models-library/src/models_library/projects_comments.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from typing import TypeAlias
+
+from models_library.projects import ProjectID
+from models_library.users import UserID
+from pydantic import BaseModel, Extra, Field, PositiveInt
+
+CommentID: TypeAlias = PositiveInt
+
+
+class _ProjectsCommentsBase(BaseModel):
+    comment_id: CommentID = Field(
+        ..., description="Primary key, identifies the comment"
+    )
+    project_uuid: ProjectID = Field(..., description="project reference for this table")
+    user_id: UserID = Field(
+        ...,
+        description="user reference for this table",
+    )
+    content: str = Field(
+        ...,
+        description="Content of the comment",
+    )
+    created_at: datetime = Field(
+        ...,
+        description="Timestamp on creation",
+    )
+    updated_at: datetime = Field(
+        ...,
+        description="Timestamp with last update",
+    )
+
+
+class ProjectsCommentsDB(_ProjectsCommentsBase):
+    class Config:
+        extra = Extra.forbid
+
+
+class ProjectsCommentsAPI(_ProjectsCommentsBase):
+    class Config:
+        extra = Extra.forbid
+        # Currently, the DB and API is one to one and is already validated when coming from the DB
```

Comment:
here I wonder if it makes any sense to validate from the DB?
You have to validate what comes from the outside. that is the API used by the frontend. to the DB and from the DB you know exactly what you save and restore. does it make sense to validate at all?

Reply:
Yes, you are right thanks, probably doesn't make sense, I will remove the validation.